### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by TonyP999**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -33,38 +39,81 @@
...
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-install-plugin</artifactId>
-        <version>2.4</version>
+        <version>2.5.2</version>
+        <executions>
+          <execution>
+            <phase>clean</phase>
+            <goals>
+              <goal>install-file</goal>
```
Comment (shoggy) [inline pom.xml:52]: 👍 for `install`

File: src/test/java/hw1/SumCalculatorTest.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import hw1.common.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import utils.CalculatorDataProvider;
+
+public class SumCalculatorTest extends BaseTest {
+
+    @Test(groups = "sumGroup", dataProvider = "sumData",
+            dataProviderClass = CalculatorDataProvider.class)
+    public void sumTest(double a, double b, double result) {
```
Comment (shoggy) [inline src/test/java/hw1/SumCalculatorTest.java:12]: `Result` is ambiguous name, explicit `actual` \ `expected` \ `expectedResult` better show your intentions

**Review by shoggy — COMMENTED** [review]
Good job!

File: src/test/java/hw1/SumCalculatorTest.java (hunk 2 above)
Comment (TonyP999) [reply src/test/java/hw1/SumCalculatorTest.java:12]: absolutely agree, thanks!

**Pull request merged**